ci-operator/config/openshift-qe/ocp-qe-perfscale-ci/openshift-qe-ocp-qe-perfscale-ci-main__aws-4.15-nightly-x86.yaml

```diff
@@ -55,6 +55,46 @@ tests:
     - chain: openshift-qe-conc-builds
     workflow: openshift-qe-installer-aws
   timeout: 5h0m0s
+- as: sdn-to-ovn-24nodes
+  cron: 0 12 * * 6
+  steps:
+    cluster_profile: aws-perfscale-qe
+    env:
+      BASE_DOMAIN: qe.devcluster.openshift.com
+      COMPUTE_NODE_REPLICAS: "24"
+      COMPUTE_NODE_TYPE: m6i.xlarge
+      CONTROL_PLANE_INSTANCE_TYPE: m6i.xlarge
+      OVN_SDN_MIGRATION_TIMEOUT: 3600s
+      ZONES_COUNT: "3"
+    post:
+    - chain: ipi-aws-post
+    pre:
+    - chain: ipi-conf-aws
+    - ref: sdn-conf
+    - chain: ipi-install
+    test:
+    - ref: openshift-qe-cluster-density-v2
+    - ref: ovn-sdn-migration
+- as: sdn-to-ovn-120nodes
+  cron: 0 14 * * 0
+  steps:
+    cluster_profile: aws-perfscale-qe
+    env:
+      BASE_DOMAIN: qe.devcluster.openshift.com
+      COMPUTE_NODE_REPLICAS: "120"
+      COMPUTE_NODE_TYPE: m6i.xlarge
+      CONTROL_PLANE_INSTANCE_TYPE: m6i.4xlarge
```

Comment:
Are these consistent with
1 - What ROSA would deploy for control plane with so many nodes - it should deploy a m5.8xl instance afaik - 4xl is too small
2 - What instance types (m5/m6) we are using for compute across the other jobs

Reply:
>1 - What ROSA would deploy for control plane with so many nodes - it should deploy a m5.8xl instance afaik - 4xl is too small

Yes, this should have been `m5.8xl` per ROSA, based on  [slack](https://redhat-internal.slack.com/archives/C05CDC19ZKJ/p1697730442544689?thread_ts=1697728570.558319&cid=C05CDC19ZKJ) conversations these changes for self-managed were propagated from the Airflow [config](https://github.com/cloud-bulldozer/airflow-kubernetes/blob/master/dags/openshift_nightlies/config/install/aws/ovn-medium.json#L8)

>2 - What instance types (m5/m6) we are using for compute across the other jobs

Currently, the master node AWS Instance families are orchestrated based on this [logic](https://steps.ci.openshift.org/reference/ipi-conf-aws#line116) ending up choosing between m6a/m6i
In order to bring more consistency we moving towards `m6i` and a PR it is [here](https://github.com/openshift/release/pull/44600)